aks/application/variables.tf

```diff
@@ -82,24 +85,28 @@ variable "is_web" {
 
 variable "web_external_hostnames" {
   type        = list(string)
+  nullable    = true
```

Comment:
I wonder if lists should really be `nullable`, what's the difference between the default `[]` and passing in an explicit `null`?